knx/dpt/formats.go

```diff
@@ -108,3 +108,25 @@ func unpackU32(data []byte, i *uint32) error {
 
 	return nil
 }
+
+func packV32(i int32) []byte {
+	b := make([]byte, 5)
+
+	b[0] = 0
+	b[1] = byte((i >> 24) & 0xff)
+	b[2] = byte((i >> 16) & 0xff)
+	b[3] = byte((i >> 8) & 0xff)
+	b[4] = byte(i & 0xff)
```

Comment:
Have you tested this for corner cases like `-1` and `MinInt32`?

Reply:
Will check. Btw. Why do we get 5 byte here (same for float type DPT9.xxx) with the first byte always zero (as far as I observed) while the datatype is just 4 byte?

Reply:
Because the KNX/net protocol is a leaky abstraction. The 2 most significant bits of the first byte are reserved. They encode the two least significant bits of the ACPI command (group read, group write, etc) once the entire frame is assembled.

In the case of V32 for instance, you actually have 38bits for data.

The reason this is also present in this package is that it gives us a way to differentiate between 6bits of data (1byte) and >6bits (>1byte).

Reply:
Added testing of corner-cases (MinInt32, MaxInt32) to unit-tests. Furthermore, all unit-test for DPT13.xxx test for both positive and negative random samples.
However, I cannot test with negative real-world data as I only do own sensors that submit positive DPT13.xxx values.